lua/entities/gmod_wire_expression2/core/string.lua

```diff
@@ -502,3 +502,69 @@ e2function string string:matchFirst(string pattern, position)
 		return Ret or ""
 	end
 end
+
+/******************************************************************************/
+local unpack = unpack
+local isnumber = isnumber
+local utf8_len = utf8.len
+
+__e2setcost(1)
+
+--- Returns the UTF-8 string from the given Unicode code-points.
+e2function string toUnicodeChar(...)
+	local args = { ... }
+	local count = #args
+	if count == 0 then return "" end
+	local codepoints = {}
+	for i = 1, count do
+		local value = args[i]
+		if typeids[i] == "n" and isnumber(value) then
+			value = math_floor(value)
+			if 0 <= value and value <= 0x10FFFF then
+				codepoints[#codepoints + 1] = value
+			end
+		end
+	end
+	self.prf = self.prf + #codepoints * 0.001
```

Comment:
Probably should use count here instead of #codepoints.

Reply:
`codepoints` table is completely sequential, there's no point in calling `table.Count` on the `codepoints` table, it would be a complete waste. Using `#` length operator is the right thing here, and it is also faster.

Reply:
I think @TomyLobo meant use the `count` variable as it's the true amount of data this function went over. Otherwise someone could pass in many variables that don't add to the `codepoints` table.

Reply:
Yeah, right...
I wanted to make a limit, but @thegrb93 said not a good idea, "it's not that bad"

But, while that would make sense, it kinda doesn't fit because in order to pass invalid type variables, they must be created somehow (and you forgot that such operation already costs too).
*Only pay for the cool icecream, not melted one.*

Reply:
You know that you can reuse an array?
Basically with your function, you could now have a chip that each tick first calls your function and then spends the rest of its ops extending the array.